cmake/modules/Findlibusb.cmake

```diff
@@ -92,7 +92,7 @@ elseif (WIN32 OR (EXISTS "/etc/debian_version" AND MINGW))
             message(STATUS "downloading libusb ${LIBUSB_WIN_VERSION}")
             file(DOWNLOAD
                 https://sourceforge.net/projects/libusb/files/libusb-1.0/libusb-${LIBUSB_WIN_VERSION}/libusb-${LIBUSB_WIN_VERSION}.7z/download
-                ${LIBUSB_WIN_ARCHIVE_PATH} EXPECTED_MD5 cf3d38d2ff053ef343d10c0b8b0950c2
+                ${LIBUSB_WIN_ARCHIVE_PATH} EXPECTED_MD5 5c944b1c8aa9d43e026a94302d0f8ac4
                 )
         endif ()
 
```

Comment:
This should be rolled back until one can be sure of that `libusb` and `mingw` work together flawlessly.
The rest of this PR seems to be fine, but please give it another test locally.

Reply:
Why do you want to accept this PR before a new release?
Let it be as a draft some time. Do new release without it.

Reply:
It's because of the codebase - it needs to be completely merged for at least three reasons:
- Merge conflict
- Different state of functionality of `master` and `develop` on at least two issues which are actively discussed.
- I have already closed a PR that partially included the content of this one. It's not a good practise to continue like this.

Thus we will not be able to merge it later due to it's history which would then reintroduce old problems from `develop` again.